package.json

```diff
@@ -24,56 +24,56 @@
   },
   "dependencies": {
     "async": "0.2.7",
-    "aws-sdk": "^2.2.4",
+    "aws-sdk": "^2.23.0",
     "bestlang": "0.0.1",
-    "body-parser": "^1.13.1",
-    "chokidar": "^1.4.2",
+    "body-parser": "^1.17.1",
+    "chokidar": "^1.6.1",
     "colors": "^1.1.2",
-    "compression": "^1.5.0",
-    "cookie-parser": "^1.3.5",
+    "compression": "^1.6.2",
+    "cookie-parser": "^1.4.3",
     "cookie-session": "^1.1.0",
     "cookies-js": "~1.2.1",
-    "cors": "^2.7.1",
-    "cryptr": "^1.0.0",
-    "csurf": "^1.8.3",
-    "express": "^4.13.0",
+    "cors": "^2.8.1",
+    "cryptr": "^2.0.0",
+    "csurf": "^1.9.0",
+    "express": "^4.15.2",
     "form-data": "^0.2.0",
-    "fs-writefile-promise": "1.0.3",
-    "glob": "^6.0.4",
+    "fs-writefile-promise": "2.0.0",
+    "glob": "^7.1.1",
     "grunt": "~0.4.1",
-    "grunt-cli": "^0.1.13",
-    "grunt-contrib-csslint": "0.1.2",
+    "grunt-cli": "^1.2.0",
+    "grunt-contrib-csslint": "2.0.0",
     "grunt-contrib-jshint": "^0.11.2",
-    "grunt-contrib-less": "^1.0.1",
-    "grunt-contrib-requirejs": "^0.4.4",
-    "grunt-execute": "0.1.5",
+    "grunt-contrib-less": "^1.4.0",
+    "grunt-contrib-requirejs": "^1.0.0",
+    "grunt-execute": "0.2.2",
     "grunt-git": "^0.3.4",
-    "grunt-lesslint": "^1.1.14",
+    "grunt-lesslint": "^3.2.0",
```

Comment:
can we keep this at `^1.1.14` and file a separate issue for grunt-lesslint specifically? Going up by two major version numbers is worth "doing on its own"